### PR TITLE
Add release documentation

### PR DESCRIPTION
**Pull request by timidri**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Releasing the dellemc-powerstore module to the Puppet Forge
+
+1. Verify that the module contains an updated [CHANGELOG.md] entry for the new release.
+1. Make sure the latest pull requests has been approved and merged.
```
Comment (AronAtDell) [inline RELEASING.md:4]: Change the numbering as they all reference "1."

Comment (timidri) [reply RELEASING.md:4]: @AronAtDell That's just how Markdown works - please see the rich preview here: https://github.com/puppetlabs/dellemc-powerstore/blob/6bee78c27923557687e4bfe7407a8507bbe88b13/RELEASING.md

Comment (AronAtDell) [reply RELEASING.md:4]: Hey Dimitri, when I brought this up in my vscode IDE it did not show the correct numbering. I am all set.

**Review by AronAtDell — APPROVED** [review]

File: RELEASING.md (hunk 1 above)
Comment (timidri) [reply RELEASING.md:4]: @AronAtDell yes, the editor doesn't always render the numbering in VS Code, but if you open a Markdown Preview window, you should see the correct Markdown rendering.

**Pull request merged**